pkg/cmd/cli/cmd/request_project.go

```diff
@@ -113,23 +116,36 @@ func (o *NewProjectOptions) Run() error {
 		return err
 	}
 
-	if o.ProjectOptions != nil {
-		o.ProjectOptions.ProjectName = project.Name
-		o.ProjectOptions.ProjectOnly = true
-		o.ProjectOptions.SkipAccessValidation = true
+	if !o.SkipConfigWrite {
+		if o.ProjectOptions != nil {
```

Comment:
I think that makes sense. I'm using `ProjectOptions` to get the server host and be able to display `...created on server https://foo`, but I think it can be taken from `o.Client`. Let me give it a try.